pkg/cidata/cidata.go

```diff
@@ -156,6 +156,9 @@ func GenerateISO9660(instDir, name string, y *limayaml.LimaYAML, udpDNSLocalPort
 			options += ",nofail"
 		}
 		args.Mounts = append(args.Mounts, Mount{Tag: tag, MountPoint: mountPoint, Type: fstype, Options: options})
+		if f.Location == "~" {
```

Comment:
```suggestion
		if Expand(f.Location) == Expand("~") {
```
(pseudo code, needs to check err)